### ISSUE TITLE
MOF compiler: Qualifier scope issue is raised as AssertionError

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#2458]
One of the recent commits to pywbem causes pywbemtools tests to fail:
* pywbemtools master (dcc2c67) on 2020-09-26 succeeds: [Travis run](https://travis-ci.org/github/pywbem/pywbemtools/jobs/730445748)
* pywbemtools PR https://github.com/pywbem/pywbemtools/pull/765 on 2020-09-30 fails: [Travis run](https://travis-ci.org/github/pywbem/pywbemtools/jobs/731487334)

The only change in pywbemtools between these two runs is in a test module (that succeeds in both runs): https://github.com/pywbem/pywbemtools/pull/765/files. pywbemtools currently uses the master branch of pywbem as a dependency.

It is therefore suspected that a pywbem change between 9/26 and 9/30 causes this. The pywbem tests succeed on its master branch, so they do not catch that error.

These pywbem changes are (new to old):
* commit 6fe4bd0: Add pylint comment to lines with pdb.set_trace()
* commit fd59a63: Add validity test to pywbem_mock CreateClass <- changes to _mof_compiler module
* Cleaned up the help formatting of mof_compiler and other scripts
* Added tests for Iter operation
* Add more tests for pywbem_mock 
* commit 3200d45 / PR #2452: Fixes issue #2451- incomplete test for qualifiers <- introduces the error

**Comment by andy-maier** [comment]
Since https://github.com/pywbem/pywbemtools/issues/766 fixes the immediate error by correcting the scope definition for the Indication qualifier in the pywbemtools test cases, the priority label can be removed from this issue, and the remaining issue for pywbem is to handle a scope violation in the input MOF better than raising an AssertionError. I have updated the title of the issue accordingly.

**Comment by andy-maier** [comment]
Since https://github.com/pywbem/pywbemtools/issues/766 fixes the immediate error by correcting the scope definition for the Indication qualifier in the pywbemtools test cases, the priority label has been removed again from this issue.

The remaining issue for pywbem is to handle a scope violation in the input MOF better than by raising an AssertionError. I have updated the title of this issue accordingly.
